site/content/en/docs/contrib/testing.en.md

```diff
@@ -6,17 +6,26 @@ description: >
   How to run tests
 ---
 
-### Unit Tests
+## Prerequisites
+
+* A recent Go distribution (>=1.12)
```

Comment:
This should be the go verison in our go.mod

Reply:
Make sense and it could be changed later too. Thus I just show link for master branch's go.mod

Reply:
Also, the go version in https://minikube.sigs.k8s.io/docs/contrib/building/iso/ and https://minikube.sigs.k8s.io/docs/contrib/building/binaries/ should be changed as similar.